src/components/InputButton/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Input, Button } from '@ht6/react-ui';
+import { ReactNode } from 'react';
+import cx from 'classnames';
+import { container, field, button } from './InputButton.module.scss';
+
+export interface InputButtonProps {
+  className?: string;
+  label: string;
+  name: string;
+  status?: {
+    state: "success" | "error";
+    text?: ReactNode;
+  };
+  onSubmit: (event: any) => any;
+  onChange: (event: any) => any;
+}
+
+function InputButton({
+  className,
+  label,
+  name,
+  status,
+  onSubmit,
+  onChange
+}: InputButtonProps) {
+  return (
+    <form onSubmit={onSubmit}>
```

Comment:
Ideally give it dynamic props since this is good for JS forms, but we lack the ability to give it key attributes for HTML only form such as method, action, etc.

As nice as JS is, we should keep in mind to put web first

Reply:
Make sure to test that `InputButtonProps` properly inherit the types of an `HTMLFormElement`